code/js/controllers/DuckburgRadioController.js

```diff
@@ -0,0 +1,18 @@
+;(function() {
+  "use strict";
+
+  var BaseController = require("BaseController");
+
+  new BaseController({
+    siteName: "Duckburg Radio",
+
+    buttonSwitch: true,
```

Comment:
I read it again now, you are right, but i didn't understand it the first time, thank you.